cscs-checks/tools/profiling_and_debugging/README.md

```diff
@@ -0,0 +1,19 @@
+This directory contains scripts to test HPC debugging and performance tools on
+Piz Daint using ReFrame. More checks are available here:
+http://github.com/eth-cscs/hpctools
```

Comment:
What I have not understood is which tools test go to the hpctools and which here.

Reply:
I'll move as many as possible to the new repo